src/lib/openjp2/j2k.c

```diff
@@ -50,7 +50,7 @@
 /*@{*/
 
 /**
- * Sets up the procedures to do on reading header. Developpers wanting to extend the library can add their own reading procedures.
+ * Sets up the procedures to do on reading header. Developers wanting to extend the library can add their own reading procedures.
```

Comment:
@rouault, you could use the same change in your GitHub profile.

Reply:
thanks :-) Frenchism corrected